Example-iOS-Swift/WebViewController.swift

```diff
@@ -17,15 +18,53 @@ class WebViewController: UIViewController, WKScriptMessageHandler {
     var avatarUrlDelegate:WebViewDelegate?
     var webView: WKWebView!
     let cookieName = "rpm-uid"
-    
+    var subscriptionCreated = false
+    //Update to your subdomain URL here
+    let subdomain = "demo"
     //Update to your custom URL here
-    let readyPlayerMeUrl = URL(string: "https://demo.readyplayer.me/avatar")!
     
-    let source = "window.addEventListener('message', function(event){ document.querySelector('.content').remove(); setTimeout(() => {window.webkit.messageHandlers.iosListener.postMessage(event.data);}, 1000) });"
+    let source = """
+            window.addEventListener('message', function(event){
+                const json = parse(event)
+        
+                if(event.data.includes('.glb')){
```

Comment:
Can it be that the data of V2 message contains this substring?

Reply:
Hmm good point let me check. I did wonder that myself but it worked so I didn't investigate further

Reply:
updated